routing/src/main/scala/routing/clients/JamClient.scala

```diff
@@ -0,0 +1,29 @@
+package routing.clients
+
+import cats.implicits.{toFlatMapOps, toFunctorOps}
+import sttp.client4.testing.SyncBackendStub.send
+import sttp.client4.{UriContext, asString, basicRequest}
+import zio._
+
+import io.circe.parser._
+import io.circe.generic.auto._
+
+class JamClient(baseUrl: String) {
+  def getJam(id: Int): ZIO[Any, Throwable, Int] = {
+    val request = basicRequest
+      .get(uri"$baseUrl/jam/$id")
+      .response(asString)
```

Comment:
Мы обычно так делаем, чтобы не париться с either на выходе
```suggestion
      .response(asStringAlways)
```